### ISSUE TITLE
Error page displayed for exception 1639821269 (valid exception)

### ISSUE THREAD

**Issue by sypets** [TYPO3-Documentation/TYPO3CMS-Exceptions#112]
## Problem

When an exceptions is thrown, the link to the exception page is displayed. When I followed it, i got a page with misleading text: 

> An Error occurred.
> **Most likely the exception you are trying to access is not a valid TYPO3 Core exception but rather caused by a 3rd-party extension.**
> ...

https://docs.typo3.org/typo3cms/exceptions/main/en-us/Exceptions/Index.html

but it is a valid TYPO3 exception for TYPO3 v12 (see below).

The URL for the exception is: https://docs.typo3.org/typo3cms/exceptions/main/en-us/Exceptions/1639821269.html

## Context of exception

vendor/typo3/cms-fluid/Classes/ViewHelpers/FlashMessagesViewHelper.php line 142:

```
// Throw if not an extbase request
                throw new \RuntimeException(
                    'ViewHelper f:flashMessages needs an extbase Request object to resolve the Queue identifier magically.'
                    . ' When not in extbase context, set attribute "queueIdentifier".',
                    1639821269
                );

```

**Comment by sbuerk** [comment]
Saw this to late. The reason for this is, that the dedicated system/repo/tool
which read the TYPO3 core exceptions per release tags and create missing files
this repo stopped working quite a while ago. No (active) people was aware of
that.

The tool is to uploaded with features not really needed. After a talk with @linawolf 
we decided to strip that down to a cli tool and integrate that directly into this
repository. And having a smaller and more maintanable tool doing it, without a dedicated
deploy host.

Sadly, I already created a dedicated issue #128 describing that and a connected PR.

**Comment by sypets** [comment]
If this is a general problem and not related to this specific exception, we could close here in favor of the other issue https://github.com/TYPO3-Documentation/TYPO3CMS-Exceptions/issues/128

ok?

**Comment by sypets** [comment]
with merge request https://github.com/TYPO3-Documentation/TYPO3CMS-Exceptions/pull/129, an exception page for **1639821269** was not created.

URL: https://docs.typo3.org/typo3cms/exceptions/main/en-us/Exceptions/1639821269.html

The exception can be found in the TYPO3 core latest main version (also in TYPO3 v12).

The exception is displayed with listExceptionCodes:

`Build/Scripts/runTests.sh -s listExceptionCodes | grep 1639821269`

**Comment by linawolf** [comment]
@sbuerk could you have a look at this please?

**Comment by sbuerk** [comment]
Yes, the initial patch have been merged. But the action did not succeeded yet. 
I'm having a eye (the workflow file could not have been tested beforehand). 

I/We currently checking that the workflow can be run, but give us there some time.
We are on it. (What are one or two more days)

**Comment by sbuerk** [comment]
@sypets The mising exception code page is now there (created from the template).

![Auswahl_1451](https://github.com/TYPO3-Documentation/TYPO3CMS-Exceptions/assets/1453466/7c8278e3-b214-4490-8f0d-dd2a4e51dc41)


Closing this issue now as fixed. Feel free to reopen it if there is still something I missed.

**Comment by sypets** [comment]
@sbuerk thank you!
